R/bm-write-csv.R

```diff
@@ -0,0 +1,74 @@
+#' Benchmark CSV writing
+#'
+#' @section Parameters:
+#' * `source` A CSV file path to write to
+#' * `writer` One of `c("arrow", "data.table", "vroom", "readr",)`
+#' * `input` One of `c("arrow_table", "data_frame")`
+#'
+#' @export
+write_csv <- Benchmark(
+  "write_csv",
+  setup = function(source = names(known_sources),
+                   writer = c("arrow", "data.table", "vroom", "readr", "utils"),
+                   input = c("arrow_table", "data_frame")) {
```

Comment:
Oh and one more thing: should we add `compression` here too? at least arrow can write compressed csvs. I believe most (all?) of the others can too, though some of the docs don't make that super obvious